lib/framebuffer.cc

```diff
@@ -112,6 +112,62 @@ class DirectRowAddressSetter : public RowAddressSetter {
   int last_row_;
 };
 
+// The SM5266RowAddressSetter (ABC Shifter + DE direct) sets bits ABC using a 8 bit shifter and DE directly.
+// The panel this works with has 8 SM5266 shifters (4 for the top 32 rows and 4 for the bottom 32 rows).
+// DE is used to select the active shifter (rows 1-8/33-40, 9-16/41-48, 17-24/49-56, 25-32/57-64).
+// Rows are enabled by shifting in 8 bits (high bit first) with a high bit enabling that row.
+// This allows up to 8 rows per group to be active at the same time (if they have the same content), but that isn't implemented here.
+// BK, DIN and DCK are the designations on the SM5266P datasheet.  
+// BK = Enable Input, DIN = Serial In, DCK = Clock
+class SM5266RowAddressSetter : public RowAddressSetter {
+public:
+  SM5266RowAddressSetter(int double_rows, const HardwareMapping &h)
+    : row_mask_(h.a | h.b | h.c | h.d | h.e), 
```

Comment:
Here, you only need h.a, h.b and h.c. The h.d and h.e we decide later if we need them in line 132.

Reason why we do this is that we then only claim the GPIO ports we need to control the library, while the remaining pins stay free for the user to use for something else.

I'll fix it.